census/controllers/api.js

```diff
@@ -151,7 +169,8 @@ var places = function (req, res) {
   }).catch(console.trace.bind(console));
 };
 
-var entries = function (req, res) {
+var entries = function (req, res, next) {
```

Comment:
To show 404 page when strategy is set but is not `cascade` nor `all`. If you have other ideas to do that - please tell me.